exo-clean/features/clean.feature

```diff
@@ -0,0 +1,42 @@
+Feature: cleaning dangling Docker images
+
+  As an Exosphere developer
+  I want to have an easy way to clean up dangling <none>:<none> Docker
+  images on my developer machine created by an Exosphere application
+  So that I don't end up with a lot of irrelevant Docker images after 
+  I build my application multiple times.
+
+  Rules:
+  - run "exo clean" in the terminal in any directory to remove dangling
+    <none>:<none> Docker images and volumes on your machine
+  - this command does not remove non-dangling Docker images/volumes
+
+
+  Scenario: cleaning a machine with both dangling and non-dangling Doker images
+    Given a running "running" application
+    When adding a file to the "users" service
+    When setting up "running" application again
```

Comment:
So there's 3 types of steps: Given, When, Then. If you use the same type of step multiple times in a row, you change the first word to `And`. For instance:
```
Given a running "running" application
When adding a file to the "users" service
And setting up "running" application again
Then ...
```